projects/data-grid/src/lib/models/data-grid-column.ts

```diff
@@ -58,14 +59,16 @@ const isFilterDefinedAndApplied = R.allPass([columnFilterDefined, R.compose(filt
 type findDataGridColumnsWithFilters = (columns: DataGridColumnWithId[]) => DataGridColumnWithId[];
 export const findDataGridColumnsWithFilters: findDataGridColumnsWithFilters = R.filter(isFilterDefinedAndApplied);
 
-export const getRawValueResolver = ({field}: DataGridColumnWithId): ColumnValueGetter => R.prop(field);
+export const getRawValueResolver = ({field}: DataGridColumnWithId): ColumnValueGetter => isString(field)
+  ? R.prop(field)
+  : R.path(field);
```

Comment:
What happens if the field or its parent is null or undefined?

Reply:
In that case the raw value resolver will return 'undefined'.